rom/dev/src/flow/cold_reset/fw_processor.rs

```diff
@@ -304,6 +304,32 @@ impl FirmwareProcessor {
                         resp.populate_chksum();
                         txn.send_response(resp.as_bytes())?;
                     }
+                    CommandId::GET_IDV_CSR => {
+                        let mut request = MailboxReqHeader::default();
+                        Self::copy_req_verify_chksum(&mut txn, request.as_bytes_mut())?;
+
+                        let csr_persistent_mem = &persistent_data.idevid_csr;
+                        let mut resp = GetIdevIdCsrResp::default();
+
+                        if csr_persistent_mem.is_unprovisioned() {
+                            /* CSR was never written to DCCM. This most likely means the
```

Comment:
nit: are there other reasons it wouldn't be populated? if not we may want to make the language in the comment stronger.